resources/views/hardware/view.blade.php

```diff
@@ -85,9 +85,8 @@
             <div class="col-md-12">
                 <div class="alert alert-danger">
                     <i class="fas fa-exclamation-triangle faa-pulse animated" aria-hidden="true"></i>
-                    <strong>WARNING: </strong>
-                    This asset has been deleted.
-                    You must restore it before you can assign it to someone.
+                    <strong>{{ trans('general.'notification_warning') }} </strong>
```

Comment:
Whoops - almost there! One too many apostrophes here :) 

`{{ trans('general.'notification_warning') }}`

should be 

`{{ trans('general.notification_warning') }}`

The entire key gets enclosed, not the segments.